src/beast/site_scons/site_tools/VSProject.py

```diff
@@ -583,6 +586,43 @@ def _guid(seed, name = None):
     guid = "{%s-%s-%s-%s-%s}" % (d[:8], d[8:12], d[12:16], d[16:20], d[20:32])
     return guid
 
+def _rmExeMod(file_name):
+    if platform.system() != 'Windows':
+        st = os.stat(file_name)
+        os.chmod(file_name, st.st_mode & ~0111)
+        return
+    
+    import win32security
+    import ntsecuritycon
+
+    user, domain, type = win32security.LookupAccountName ("", "Everyone")
```

Comment:
`type` is a built-in function - and see that github knows this too and mis-colors this.  While it's legal to overwrite built-in functions, and it's only happening locally, I suggest not doing that.

Also - quibble, you were using ``'` above, might as well stick with it.  (Plus you don't have to hold down the shift key to type `'`.)